lib/writers/git.go

```diff
@@ -219,7 +219,7 @@ func (g *GitWriter) ReadFile(filePath string) ([]byte, error) {
 	defer os.RemoveAll(filepath.Dir(localTmpDir))
 
 	if _, err := worktree.Filesystem.Lstat(fullPath); err != nil {
-		logger.Error(err, "could not stat file")
+		logger.Info("could not stat file", "err", err)
```

Comment:
sometimes this is fine, so having this error output was making me think the reconciler exited when it hasn't. the cosnume of this detects if it should error log or ignore